packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -20,7 +20,7 @@ $Rating-largeIconSize: 20px;
   display: inline-block;
 
   &:hover {
-    .star:not(.starIsDisabled) {
+    .star:not(.starIsDisabled):not(.starIsAggregate):not(.partialStar) {
```

Comment:
This seems gross. a disabled rating component should be disabled, not conditionally.